interfaces/about.html

```diff
@@ -0,0 +1,216 @@
+          <div class="hero-content mx-auto">
+            <p class="red-text">"Welcome To DevBox!!"</p>
+            <h1 class="red-heading">
+              The Global Conference For Software Engineers 2022
+            </h1>
+            <div class="bg-secondry border border-white p-2 my-4">
+              <p>
+                Global Software Engineering and Technology Conference aims to
+                bring together leading academic scientists, researchers and
+                research scholars to exchange and share their experiences and
+                research results on all aspects of Global Software Engineering
+                and Technology Conference. It also provides a premier
+                interdisciplinary platform for researchers, practitioners, and
+                educators to present and discuss the most recent innovations,
+                trends, and concerns as well as practical challenges encountered
+                and solutions adopted in the fields of Global Software
+                Engineering and Technology Conference
+              </p>
+            </div>
+            <p>Contact us per Email for any further questions</p>
+            <a href="#">devbox@gmail.com <span>&#10148;</span></a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="logo-section">
+      <div class="container-fluid">
+        <div class="container-md text-center">
+          <h2>DevBox Software Development Logo</h2>
+          <hr class="separator" />
+          <p class="col-lg-6 mx-auto">
+            The Logo of DevBox Software Engineers Conference 2022 was dicided
+            throught the logo compitition from 20 Jun to 7 July.
+          </p>
+          <div
+            class="logo-container col-12 col-md-6 d-flex justify-content-center border p-5 mx-auto mt-4"
+          >
+            <img src="../assets/logo/Logo.svg" alt="Logo" />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <hr />
+
+    <section id="past">
+      <div class="container-fluid">
+        <div class="container-md text-center">
+          <h2>See the past DevBox Global Conferences</h2>
+          <hr class="separator" />
+          <p>
+            Take a look at DevBox Global Conferences which took place in England
+            and France
+          </p>
+          <div class="past-imgs-container row">
+            <div class="col-12 col-lg-6">
+              <div class="past-conf-1 mb-3 mb-lg-0">
+                <strong>2015</strong>
+                <p>DevBox Global 2015 in France</p>
+              </div>
+            </div>
+            <div class="col-12 col-lg-6">
+              <div class="past-conf-2">
+                <strong>2018</strong>
+                <p>DevBox Global 2018 in England</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- PARTNER SECTION -->
+    <section id="partners" class="bg-dark d-md-none">
```

Comment:
- [ ] Kindly add a partners section also for a desktop screen on the about page. Currently, it's only visible in the mobile version.
![image](https://user-images.githubusercontent.com/92215878/190701769-1ca5c667-5d65-4769-ae6e-978063962753.png)